pkg/storageincentives/agent.go

```diff
@@ -244,11 +244,11 @@ func (a *Agent) start(blockTime time.Duration, blocksPerRound, blocksPerPhase ui
 		// compute the current phase
 		p := block % blocksPerRound
 		if p < blocksPerPhase {
-			currentPhase = commit // [0, 37]
-		} else if p >= blocksPerPhase && p < 2*blocksPerPhase { // [38, 76]
+			currentPhase = commit // [0, 38)
```

Comment:
These range comments are incorrect.  A given block cannot be in multiple phases.   commit is 0-37 (< blocks/4), reveal is 38-75 (< blocks/2) and claim is 76-151.  There is no 152 remainder when doing a mod on 152.    The code matches the newly deployed contract, but the comments don't match the code (unless you go with non-inclusive on the right).

From the contract at https://goerli.etherscan.io/address/0x2e4ccEb9a88bcfbC7779Fa693714783A345c1343#code
```
    function currentPhaseCommit() public view returns (bool){
        if ( block.number % roundLength < roundLength / 4 ) {
            return true;
        }
        return false;
    }

    function currentPhaseReveal() public view returns (bool){
        uint256 number = block.number % roundLength;
        if ( number >= roundLength / 4 && number < roundLength / 2 ) {
            return true;
        }
        return false;
    }

    function currentPhaseClaim() public view returns (bool){
        if ( block.number % roundLength >= roundLength / 2 ) {
            return true;
        }
        return false;
    }
```

Reply:
hi @ldeffenb, thanks for the comment. 

The reveal phase had 39 blocks in the previous iteration of the contract. 
We redeployed the contract with some fixes, which includes making every phase the same length. 

But yes, if we hadn't changed the contract, you are absolutely correct.

Reply:
The upcoming release will have the new contract addresses.

Reply:
commit is inclusive 0, exclusive 38 [0,38)
reveal is inclusive 38, exclusive 76 [38, 76)
claim is inclusive 76, exclusive 152 [76, 152)

in other words, [0, 37], [38, 75], [76, 151]

maybe I am missing something, but this looks correct and is aligned with the code and the new contract.

Reply:
The comments say the following:
commit: 0, 38
reveal: 38, 76
claim: 76, 152
Which is correct if they are going non-inclusive on the end value.   But to a first glance, it appears that blocks offset 38 and 76 are actually in two phases when they are not.  And it also implies that the % (mod) function can return 152, which it cannot.  
I'm suggesting that comments reflect the actual round/block offsets of:
commit: 0, 37 (38 blocks)
reveal: 38, 75 (38 blocks)
claim: 76, 151 (76 blocks)
Possibly even adding the parentheticals for 100% clarity to the reader.